src/DevBetterWeb.UploaderApp/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Ardalis.ApiCaller;
+using DevBetterWeb.Vimeo.Models;
+using DevBetterWeb.Vimeo.Services.VideoServices;
+
+namespace DevBetterWeb.UploaderApp
+{
+  class Program
+  {
+    static async Task Main(string[] args)
+    {
+      var argsList = args.ToList();
+      if (argsList.Count == 0 || argsList.All( x => x.ToLower() != "-d") || argsList.All(x => x.ToLower() != "-t"))
+      {
+        Console.WriteLine("PLease use -d [destination folder] -t [Vimeo token]");
+        return;
+      }
+
+      var folderToUploadIndex = argsList.FindIndex(x => x.ToLower() == "-d") + 1;
+      if(folderToUploadIndex <= 0)
+      {
+        Console.WriteLine("PLease use -d [destination folder]");
+        return;
+      }
+
+      var tokenIndex = argsList.FindIndex(x => x.ToLower() == "-t") + 1;
+      if (tokenIndex <= 0)
+      {
+        Console.WriteLine("PLease use -t [Vimeo token]");
+        return;
+      }
+
+      var folderToUpload = argsList[folderToUploadIndex];
```

Comment:
Ideally once you've gathered and validated the arguments to the program, you should create and call out to a service. This lets you better unit test the service, since it's not necessarily easy to test a console main() method.

Reply:
UploaderManager created.